annotation/app/models.py

```diff
@@ -332,3 +359,51 @@ def get_stats(self):
             "time_finish": self.task.stats.updated,
             "agreement_score": self.agreement_score,
         }
+
+
+class DocumentLinks(Base):
+    __tablename__ = "document_links"
+
+    original_revision = Column(VARCHAR)
+    original_file_id = Column(INTEGER)
+    original_job_id = Column(INTEGER)
+
+    similar_revision = Column(VARCHAR)
+    similar_file_id = Column(INTEGER)
+    similar_job_id = Column(INTEGER)
+
+    label = Column(VARCHAR, nullable=False)
+
+    __table_args__ = (
+        ForeignKeyConstraint(
+            (original_revision, original_file_id, original_job_id),
+            (AnnotatedDoc.revision, AnnotatedDoc.file_id, AnnotatedDoc.job_id),
+        ),
+        PrimaryKeyConstraint(
```

Comment:
We need to add similar doc as primary key as well. Since it is possible to have one to many link here.

Reply:
You are right, missed this. Added similar doc as primary key+